mobile/src/screens/chat-layout/components/chat-input/chat-input.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  type Control,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+
+import SendButton from '#assets/img/icons/send-button.svg';
+import { Pressable, TextInput, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { useFormController } from '#libs/hooks/hooks';
+
+import { styles } from './styles';
+
+type Properties<T extends FieldValues> = {
+  name: FieldPath<T>;
+  control: Control<T, null>;
+  scrollViewToEnd: () => void;
+  onPress: () => void;
+};
+
+const ChatInput = <T extends FieldValues>({
+  name,
+  control,
+  scrollViewToEnd,
+  onPress,
+}: Properties<T>): JSX.Element => {
+  const { field } = useFormController({ name, control });
+  const { value, onChange } = field;
+
+  return (
+    <View style={styles.chatInputContainer}>
```

Comment:
let's simplify class name
```suggestion
    <View style={styles.container}>
```